recipes/cpc2/baseline/evaluate.py

```diff
@@ -0,0 +1,73 @@
+"""Evaluate the predictions against the ground truth correctness values"""
+import json
+import logging
+from pathlib import Path
+
+import hydra
+import numpy as np
+import pandas as pd
+from omegaconf import DictConfig
+from scipy.stats import kendalltau, pearsonr
+
+logger = logging.getLogger(__name__)
+
+
+def rmse_score(x: np.ndarray, y: np.ndarray) -> float:
```

Comment:
Could this, and other functions, be grouped under a module in `clarity/stats.py` perhaps? I recall seeing the Root Mean Square Error and other statistics perhaps being calculated in many places throughout the code base and so abstracting out would reduce duplication.

Appreciate the release is looming if not overdue and this is a big PR so perhaps more appropriate to make it a separate issue and deal with it subsequently.